CalculateAriphmeticMeanOfThreeShorts.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+public class CalculateAriphmeticMeanOfThreeShorts {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input 3 whole numbers ");
+        short Number1 = sc.nextShort();
+        short Number2 = sc.nextShort();
+        short Number3 = sc.nextShort();
+        int Mean = (Number1 + Number2 + Number3) / 3;
+        System.out.println(Mean);
+        if(Number1 > 0) {
```

Comment:
What if you only check whether the Number is negative, and only in that case do smth, without `else` here and below in all` if else` stetements.

Reply:
got it